charts/cluster-autoscaler/values.yaml

```diff
@@ -95,6 +95,9 @@ azureClusterName: ""
 # Required if `cloudProvider=azure`
 azureNodeResourceGroup: ""
 
+# azureUseWorkloadIdentityExtension -- Whether to use Azure's workload identity extension for credentials.
+azureUseWorkloadIdentityExtension: false
```

Comment:
+1 to mention here WorkloadIdentity couldn't be used together with other author approaches.

Reply:
And how `AADFederatedTokenFile` could be set in the helm chart?

Reply:
I added documentation to the readme